app/main.py

```diff
@@ -1,25 +1,69 @@
+from __future__ import annotations
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: SlideLimitationValidator, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self,
+                instance: SlideLimitationValidator,
+                owner: SlideLimitationValidator) -> None:
+        getattr(instance, self.protected_name)
+
+    def __set__(self,
+                instance: SlideLimitationValidator,
+                value: int) -> None:
+        if value not in range(self.min_amount, self.max_amount + 1):
+            raise ValueError
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, height: int, weight: int) -> None:
+        self.name = name
+        self.age = age
+        self.height = height
+        self.weight = weight
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, height: int, weight: int) -> None:
+        self.age = age
+        self.height = height
+        self.weight = weight
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    height = IntegerRange(80, 120)
+    weight = IntegerRange(20, 50)
 
 
 class AdultSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(14, 60)
+    height = IntegerRange(120, 220)
+    weight = IntegerRange(50, 120)
 
 
 class Slide:
-    pass
+    def __init__(self,
+                 name: str,
+                 limitation_class: type(SlideLimitationValidator)
```

Comment:
```suggestion
                 limitation_class: type[SlideLimitationValidator]
```
Do like this. Because before you just call function type()